### DISCUSSION TITLE
Are You looging for help?

### DISCUSSION THREAD

**Discussion by david-vega** [5axes/CustomSupportCylinder#40] [General]
I Just found this plugin and also realized that is not properly maintained anymore. I'm a developer and I can help you with that. I'm gonna need some help from you to understand the project initially. Once that's done, we can discuss the further steps. Let me know if you are interested.

**Comment by 5axes** [comment]
https://github.com/5axes/CustomSupportCylinder/discussions/39#discussioncomment-9774085
